src/v0/destinations/ga4/utils.js

```diff
@@ -552,6 +554,58 @@ const buildDeliverablePayload = (payload, Config) => {
   return response;
 };
 
```

Comment:
I was suggesting to move to entirely to new util class under GA4 `ga4/customMappingUtil.js` as this class too has become huge

Reply:
Later on this class can be moved to root util when we onboard more destinations

Reply:
Addressed